compose-profiles/dev-local-3nodes/docker-compose.yml

```diff
@@ -168,6 +168,8 @@ services:
     extends:
       file: ../docker-compose-definitions.yml
       service: postgresql
+    environment:
```

Comment:
To be removed, see comment after.

Reply:
I need this variable to make the postgresql image create the databases no?